requirements.txt

```diff
@@ -0,0 +1,11 @@
+appdirs==1.4.4
+certifi==2020.12.5
+chardet==4.0.0
+humanfriendly==9.1
+humanize==3.2.0
+hurry.filesize==0.9
+idna==2.10
+python-magic==0.4.18
+python3-wget @ git+https://github.com/bernd-wechner/python3-wget@e38979ffdf462a5cec05c5a92ff35a7630c6435b
+requests==2.25.1
+urllib3==1.26.2
```

Comment:
python-dateutil==2.8.1

Reply:
Oh yeah I was wondering what the actual module name is and I can't remember. Thanks for pointing this out to me!

Reply:
Thanks enormously!

I'd rather not see output if the configuration is right, and I'd rather see it copy the file from the build folder to config folder than hardcoding the string in the code I admit. 

Love `requirements.txt`!

One reason I keep the default properties separate is in fact for legibility, and easy replacemeny. I nabbed them from the Degoo web apps request. And they represent a conundrum alas, because there is no guarantee they are exhaustive, and I'm hoping there's more, in particular a hash of the file for sensible change detection.  

Thus far they suggested MetadataKey might be a SHA1 hash, but it's not one I can reproduce.

Reply:
Just to be honest, I’m not really well versed in Python 😅though, I will try my best not to hardcode strings in the future.

Reply:
No worries. I'm really please you're helping out!